fix_topic: /gps/fix
map_frame: map
initial_pose:
  roll: 0.0
  pitch: 0.0
  yaw: -2.76